sdks/python/apache_beam/runners/dataflow/internal/apiclient_test.py

```diff
@@ -356,6 +356,34 @@ def test_labels(self):
     self.assertEqual('key5', job.proto.labels.additionalProperties[4].key)
     self.assertEqual('', job.proto.labels.additionalProperties[4].value)
 
+  def test_experiment_use_multiple_sdk_containers(self):
+    pipeline_options = PipelineOptions(
```

Comment:
Is it possible to actually fail the pipeline if the experiments are invalid? Are we doing that? seems like the user was having issues getting the format correct. Just want to make sure we error out on invalid input

--experiment A,B .   should fail. That was the pattern we saw
Also, do we print a good error message with proper sample input?

Please fix in this PR or add a TODO with a JIRA number.

Reply:
No we do not fail this out. But "A,B" might actually be a valid experiment. This case is unlikely, but one generally possible case for having commas in the experiment is --experiment A=B,C,D